### PR TITLE
Fix (plugins/[request/response]-transformer) Adding migration scripts

### PR DESCRIPTION
**Pull request by shashiranjan84**
Added missing migration script for request/response transformer plugins. #1064.

### CODE REVIEW

File: kong/plugins/request-transformer/schema/migrations.lua [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+      local plugins, err = dao_factory.plugins:find_by_keys {name = "request-transformer"}
+      if err then
+        return err
+      end
+
+      for _, plugin in ipairs(plugins) do
+        for _, action in ipairs {"remove", "add", "append", "replace"} do
+
+          if plugin.config[action] == nil then
+            plugin.config[action] = {}
```
Comment (thibaultCha) [inline kong/plugins/request-transformer/schema/migrations.lua]: `plugin.config[action] = plugin.config[action] or {}` will be more concise and more consistent with l21

File: kong/plugins/request-transformer/schema/migrations.lua [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        return err
+      end
+
+      for _, plugin in ipairs(plugins) do
+        plugin.config.replace = nil
+        plugin.config.append = nil
+        
+        for _, action in ipairs {"remove", "add"} do
+          for _, location in ipairs {"body", "headers", "querystring"} do
+            if plugin.config[action] ~= nil and next(plugin.config[action][location]) == nil then
```
Comment (thibaultCha) [inline kong/plugins/request-transformer/schema/migrations.lua]: What does it matter if `plugin.config[action][location]` is an empty table or not? It needs to be set to `nil` regardless.

Comment (shashiranjan84) [inline kong/plugins/request-transformer/schema/migrations.lua]: Not all off them, like plugin.config.add.headers can have data and would need to be migrated(basically untouched)

**Comment by thibaultCha** [comment]
I trust that you tested this against all possible scenarios 
- up from 0.5.0 with existing data
- up from 0.7.0 with existing data
- up from 0.5.0 with empty config
- up from 0.7.0 with empty config
- down with existing data
- down with empty config
- potentially up again

Otherwise, this needs to be rebased on next, especially with the new DAO. I don't think that a postgres migration is needed though.

File: kong/plugins/request-transformer/schema/migrations.lua (hunk 1 above)
Comment (shashiranjan84) [inline kong/plugins/request-transformer/schema/migrations.lua]: Right, updating it

**Comment by shashiranjan84** [comment]
Yes I tested `UP` for all the scenarios and worked fine. Even tested `DOWN`, all worked but reponse-transformer down migration was not showing updated value in table.

**Pull request merged**